uktrade/data-hub-frontend
# Cps 519 proposition complete message

## Description of change

Show the error/completion message when completing a proposition on the overview proposition list and on the proposition details page. 
Remove the old version of posting the complete form with the task component

## Test instructions

Create a proposition, do not add a document to the proposition. Click the complete button and the error will appear

## Screenshots
<img width="1156" alt="image" src="https://github.com/uktrade/data-hub-frontend/assets/15215184/1ac9060b-5097-434b-bc03-75b3b15389e6">


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
